app/backend/src/couchers/models.py

```diff
@@ -805,12 +805,24 @@ class PageVersion(Base):
     editor_user_id = Column(ForeignKey("users.id"), nullable=False, index=True)
     title = Column(String, nullable=False)
     content = Column(String, nullable=False)
+    # the human-readable address
+    address = Column(String, nullable=False)
     geom = Column(Geometry(geometry_type="POINT", srid=4326), nullable=True)
     created = Column(DateTime(timezone=True), nullable=False, server_default=func.now())
 
-    page = relationship("Page", backref="versions")
+    page = relationship("Page", backref="versions", order_by="PageVersion.id")
     editor_user = relationship("User", backref="edited_pages")
 
+    @property
+    def coordinates(self):
+        # returns (lat, lng)
+        # we put people without coords on null island
+        # https://en.wikipedia.org/wiki/Null_Island
+        if self.geom:
```

Comment:
If we make geom required we wouldn't have to special handle NULLs in the database. We could make it required as there's no way through the proposed API to create a page without a coordinate.

Reply:
I'm not sure if all pages will have a coordinate. There will presumably be some general-purpose guides that are related to a community and only show up on the community page.

I've changes this now to return coordinates or `None` if there is none, and moved the `(0,0)` stuff out of the models, as per @amdwit's suggestion.